### ISSUE TITLE
Almost working on an515-58, the fan on the left doesnt work

### ISSUE THREAD

**Issue by p-aaditya** [JafarAkhondali/acer-predator-turbo-and-rgb-keyboard-linux-module#115]
# Model: Nitro AN515-58

### Turbo button activates overclock? Yes\No

since mine is a nitro i dont have a turbo button but i do have a nitrosense button so are they same?
next rgb backlight works so good
one of the fans seem to be working which is the one on the right when i run install so nice again
the other fan on the left dont seem to be working when i run install
so please help me in making the left fan work
thanks in advance

**Comment by p-aaditya** [comment]
isnt your playlist more towards making software communicate with rgb lights but for me the rgb lights work fine except for the gpu fan not working so how should i proceed in order to solve this problem

**Comment by JafarAkhondali** [comment]
Yes but they are similar. You can De-compile NitroSense and find out how the turbo mode function works.

**Comment by jyash8** [comment]
I have the same model. The only workaround i know is to use nbfc-linux. Just pick config of any acer nitro close to the model. I had picked this config "Acer Nitro AN515-51". It is working fine and i am able change my fan manually and auto mode is also working fine.

**Comment by p-aaditya** [comment]
> I have the same model. The only workaround i know is to use nbfc-linux. Just pick config of any acer nitro close to the model. I had picked this config "Acer Nitro AN515-51". It is working fine and i am able change my fan manually and auto mode is also working fine.

well i did try using nbfc-linux but that did not help. an515-51 config file would never change my current fan speed

**Comment by p-aaditya** [comment]
or if i can get a config file that can help ass well

> I have the same model. The only workaround i know is to use nbfc-linux. Just pick config of any acer nitro close to the model. I had picked this config "Acer Nitro AN515-51". It is working fine and i am able change my fan manually and auto mode is also working fine.

**Comment by jyash8** [comment]
Can you tell me the commands you used to try to change the fan speed

> I have the same model. The only workaround i know is to use nbfc-linux. Just pick config of any acer nitro close to the model. I had picked this config "Acer Nitro AN515-51". It is working fine and i am able change my fan manually and auto mode is also working fine.

**Comment by jyash8** [comment]
Try this command
```shell
 sudo nbfc set -f 1 -a
```

**Comment by p-aaditya** [comment]
> Try this command
> 
> ```shell
>  sudo nbfc set -f 1 -s 100
> ```
> 
> Your fan should ramp up to full speed.

i have used this command it only changes the target fan speed but the current fan speed in unaffected
